docs/KPI.md

```diff
@@ -92,7 +92,7 @@ A viewport contains the following fields:
 
 ### `annotations`
 
-(Optional) Defines thresholds for the line chart. To view and interact with an annotation example, see [Annotation](https://synchrocharts.com/#/Features/Annotation) in the Synchro Charts documentation. For more information about the `annotations` API, see [Properties](https://synchrocharts.com/#/API/Properties) in the Synchro Charts documentation. 
+(Optional) Defines thresholds for the kpi. Annotations that aren't also thresholds will be ignored. To view and interact with an annotation example, see [Annotation](https://synchrocharts.com/#/Features/Annotation) in the Synchro Charts documentation. For more information about the `annotations` API, see [Properties](https://synchrocharts.com/#/API/Properties) in the Synchro Charts documentation. 
```

Comment:
Non-blocking: Should we make capitalization consistent (KPI)?

Reply:
fixed in revision